RedditSharp/Things/LiveUpdateEvent.cs

```diff
@@ -545,7 +539,7 @@ private void CommonInit(Reddit reddit, JToken json, IWebAgent webAgent)
             WebAgent = webAgent;
         }
 
-        private void SimpleAction(string url)
+        private new void SimpleAction(string url)
```

Comment:
right, but because it's hiding that, it doesn't really behave quite like you'd expect when it's cast as it's parent type etc. The method should be marked `virtual` and instead of new it should be `overrides`